### ISSUE TITLE
[PRE REVIEW]: Controlling video stimuli in sign language and gesture research: The OpenPoseR package for analyzing OpenPose motion tracking data in R

### ISSUE THREAD

**Issue by whedon** [openjournals/joss-reviews#2748]
**Submitting author:** @trettenbrein (<a href="http://orcid.org/0000-0003-2233-6720">Patrick C. Trettenbrein</a>)
**Repository:** <a href="https://github.com/trettenbrein/OpenPoseR" target ="_blank">https://github.com/trettenbrein/OpenPoseR</a>
**Version:** v1.0
**Editor:** Pending
**Reviewer:** Pending
**Managing EiC:** Daniel S. Katz

**:warning: JOSS reduced service mode :warning:**

Due to the challenges of the COVID-19 pandemic, JOSS is currently operating in a "reduced service mode". You can read more about what that means in [our blog post](https://blog.joss.theoj.org/2020/05/reopening-joss).

**Author instructions**

Thanks for submitting your paper to JOSS @trettenbrein. **Currently, there isn't an JOSS editor assigned** to your paper.

@trettenbrein if you have any suggestions for potential reviewers then please mention them here in this thread (without tagging them with an @). In addition, [this list of people](https://bit.ly/joss-reviewers) have already agreed to review for JOSS and may be suitable for this submission (please start at the bottom of the list).

**Editor instructions**

The JOSS submission bot @whedon is here to help you find and assign reviewers and start the main review. To find out what @whedon can do for you type:

```
@whedon commands
```

**Comment by whedon** [comment]
Hello human, I'm @whedon, a robot that can help you with some common editorial tasks.

**:warning: JOSS reduced service mode :warning:**

Due to the challenges of the COVID-19 pandemic, JOSS is currently operating in a "reduced service mode". You can read more about what that means in [our blog post](https://blog.joss.theoj.org/2020/05/reopening-joss).

For a list of things I can do to help you, just type:

```
@whedon commands
```

For example, to regenerate the paper pdf after making changes in the paper's md or bib files, type:

```
@whedon generate pdf
```

**Comment by whedon** [comment]
```
Software report (experimental):

github.com/AlDanial/cloc v 1.84  T=0.39 s (1729.5 files/s, 5346.9 lines/s)
-------------------------------------------------------------------------------
Language                     files          blank        comment           code
-------------------------------------------------------------------------------
R                               18            149            344            668
JSON                           657              0              0            657
Markdown                         1             22              0             32
Rmd                              1             82            115             24
-------------------------------------------------------------------------------
SUM:                           677            253            459           1381
-------------------------------------------------------------------------------


Statistical information for the repository 'd3deb90461bbb899355b3bc3' was
gathered on 2020/10/13.
No commited files with the specified extensions were found.
```

**Comment by whedon** [comment]
PDF failed to compile for issue #2748 with the following error: 

 Can't find any papers to compile :-(

**Comment by danielskatz** [comment]
@whedon generate pdf from branch paper

**Comment by whedon** [comment]
```
Attempting PDF compilation from custom branch paper. Reticulating splines etc...
```

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/openjournals/joss-papers/joss.02748/joss.02748/10.21105.joss.02748.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/openjournals/joss-papers/blob/joss.02748/joss.02748/10.21105.joss.02748.pdf) :page_facing_up: :point_left:

**Comment by danielskatz** [comment]
@whedon check references from branch paper

**Comment by whedon** [comment]
```
Attempting to check references... from custom branch paper
```

**Comment by whedon** [comment]
```
Reference check summary (note 'MISSING' DOIs are suggestions that need verification):

OK DOIs

- 10.1111/1467-8721.ep13175642 is OK
- 10.1037/0033-295X.92.3.350 is OK
- 10.1093/oxfordhb/9780198786825.013.25 is OK
- 10.1016/j.neuropsychologia.2011.06.029 is OK
- 10.1037/0096-1523.7.2.430 is OK
- 10.1109/MMUL.2012.24 is OK
- 10.3758/s13428-012-0249-2 is OK
- 10.31234/osf.io/9nsz3 is OK
- 10.1080/23273798.2018.1499946 is OK
- 10.25592/UHHFDM.842 is OK

MISSING DOIs

- 10.1109/cvpr.2017.143 may be a valid DOI for title: Realtime Multi-Person 2D Pose Estimation Using Part Affinity Fields
- 10.1093/oxfordhb/9780199573776.013.21 may be a valid DOI for title: The Syntax of Sign Language and Universal Grammar

INVALID DOIs

- None
```

**Comment by danielskatz** [comment]
👋 @trettenbrein -

- note that there might be some missing DOIs you can add to your bib - you can check them and regenerate the paper using the same command I used above
-  because of the relatively small size of your code, the editors will discuss the scope to make sure it meets our [substantial scholarly effort](https://joss.readthedocs.io/en/latest/submitting.html#substantial-scholarly-effort) criterion before we start a review - this will likely take until the middle of next week.

**Comment by danielskatz** [comment]
@whedon query scope

**Comment by whedon** [comment]
Submission flagged for editorial review.

**Comment by trettenbrein** [comment]
> 👋 @trettenbrein -
> 
> * note that there might be some missing DOIs you can add to your bib - you can check them and regenerate the paper using the same commands I used above
> * because of the relatively small size of your code, the editors will discuss the scope to make sure it meets our [substantial scholarly effort](https://joss.readthedocs.io/en/latest/submitting.html#substantial-scholarly-effort) criterion before we start a review - this will likely take until the middle of next week.

Hi @danielskatz,

Many thank for your effort! I've updated the list of references with the two missing DOIs. @whedon generate pdf from branch paper

We're happy to wait for your editorial decision. In case you decide to send the paper for review, I'd like to suggest @cMadan as a possible editor. Also, after going through the list of reviewers you provided we'd like to suggeste that eirini-zormpa, navmayan, avakiai, rmhogervorst, janfreyberg, craddm, mmrabe, matherion may be suitable reviewers as they have all listed either linguistics, psycholinguistics, or psychology and R as part of their areas of expertise.

Best,
Patrick

**Comment by trettenbrein** [comment]
@whedon generate pdf from branch paper

**Comment by whedon** [comment]
```
Attempting PDF compilation from custom branch paper. Reticulating splines etc...
```

**Comment by arfon** [comment]
@whedon reject

@trettenbrein - many thanks for your submission to JOSS. After consulting with the editorial team, we have concluded that this submission is out of scope for JOSS due to [its limited size](https://joss.readthedocs.io/en/latest/submitting.html#substantial-scholarly-effort).

**Comment by whedon** [comment]
Paper rejected.
